docs/user-guide/install-zowe-smpe.md

```diff
@@ -712,6 +712,33 @@ IEF257I ZWE3ALOC ALLOCD ALLOCD AZWEZFS - SPACE REQUESTED NOT AVAILABLE
 IEF272I ZWE3ALOC ALLOCD ALLOCD - STEP WAS NOT EXECUTED.                    
 ```
 Uncomment the `VOL=SER=&...` control statements and refer to the comments at the start of the JCL job for related necessary changes.
+### Prepare the Zowe SMP/E Installation Jobs with Velocity Template Language (VTL) Command-line Interface or Python Script (Optional)
+The Velocity Template Language (VTL) Command-line Interface and Python Script simplifies the Zowe SMP/E installation jobs customization using the YAML file.  To install and use the Velocity Template Language (VTL) Command-line Interface, refer to https://github.com/plavjanik/vtl-cli.
```

Comment:
Need to make this a real link in Markdown. So change it to `see [https://github.com/plavjanik/vtl-cli](https://github.com/plavjanik/vtl-cli)`.